### DISCUSSION TITLE
Welcome to ASAM Discussions!

### DISCUSSION THREAD

**Discussion by AncillaryServicesAcquisitionModel** [AncillaryServicesAcquisitionModel/ASAM#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our ASAM community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage and collaborate with other community members.

**Comment by Manish-Khanra** [comment]
How does the ASAM is interoperable or coupled with PyPSA?

**Reply by SamuelGoodday** [reply]
Hi @Manish-Khanra, ASAM uses today already PyPSA in several classes. Market parties use a PyPSA model for their optimal asset portfolio dispatch. The redispatch mechansims (of the market operator class) use PyPSA model for clearing redispatch bids. What is not yet implemented is a loadflow calculation by the Grid Operator class to determine the redispatch need. Technically this is no problem given the current 'interoperability' .
